nss-day-cohort-46/rare-rest-rare-api-cool-cake
# Cr categories delete

# Description

Staff can edit tags.
Closes #16 

## Type of change


# Testing Instructions

`git fetch --all`
`git checkout cr-categories-edit`
`python3 manage runserver`


For client-side instructions, see frontend pull request.

In Postman, change Authentication token to staff user.

Make a `GET` request to `http://localhost:8000/categories/5` to view its label.

Then make a `PUT` request to `http://localhost:8000/categories/5` with the following in the body.

{
  "label": "new label"
}

then make a new `GET` request to view updated label.

 Change Authentication token to non-staff user and follow instructions above. User will receive a non-authorized error.

When logged in as staff, click on "Tag Management" and there will be an option to edit a tag.
Click on `Edit` to view tag form to update.
Change current label.
When click on `Update`, user will see tag updated.

When logged in as non-staff, `Edit` option is unavailable.

# Checklist:
